components/events/EventMenu.vue

```diff
@@ -1,7 +1,12 @@
 <template>
   <nav>
     <section class="ibm-research">
-      <a href="https://qiskit.org" class="text-logo" target="_blank" rel="noopener"><img class="logo" alt="Qiskit logo" src="https://qiskit.camp/img/qiskit-logo.png">&nbsp;Qiskit</a>
+      <AppLink
+        class="text-logo"
+        v-bind="{ url: '/'}"
```

Comment:
No need for dynamic binding here:
```suggestion
        url="/"
```